### PR TITLE
Add the Subjective Value Inventory

### PR DESCRIPTION
**Pull request by xehu**
Linked Issue: https://github.com/Watts-Lab/surveys/issues/148

### CODE REVIEW

**Comment by xehu** [comment]
Ooof, looks like all the tests failed.......weirdly, I don't see `SVI.cy.jsx` on this list?

File: cypress.config.js [hunk 1]
```diff
@@ -45,4 +46,10 @@ module.exports = defineConfig({
     // Default is 0
     openMode: 0,
   },
+
```
Comment (JamesPHoughton) [inline cypress.config.js:49]: we're only running component tests in this library at the moment, so lets revert changes to this file.

File: src/index.js [hunk 2]
```diff
@@ -84,6 +84,11 @@ import relatednessNeedSatisfactionSha from "../surveys/relatednessNeedSatisfacti
 import { default as relatednessNeedSatisfactionScoreFunc } from "../surveys/relatednessNeedSatisfaction/relatednessNeedSatisfaction.score.js";
 export const RelatednessNeedSatisfaction = SurveyFactory("relatednessNeedSatisfaction", relatednessNeedSatisfactionJson, relatednessNeedSatisfactionScoreFunc, relatednessNeedSatisfactionSha);
 
+import SVIJson from "../surveys/SVI/SVI.json";
```
Comment (JamesPHoughton) [inline src/index.js:87]: for some reason, the SVI didn't get included in the PR? Is it gitignored?

**Review by JamesPHoughton — CHANGES_REQUESTED** [review]
Looks like the survey folder didn't get added properly. If you want to try again, here's just a few suggestions. Otherwise, could have a quick chat around 3:00 today?

**Review by JamesPHoughton — CHANGES_REQUESTED** [review]
Looks like the survey folder didn't get added properly. If you want to try again, here's just a few suggestions. Otherwise, could have a quick chat around 3:00 today?

**Comment by xehu** [comment]
> Looks like the survey folder didn't get added properly. If you want to try again, here's just a few suggestions. Otherwise, could have a quick chat around 3:00 today?

Oops, for some reason it didn't push! should be there now...

File: cypress.config.js (hunk 1 above)
Comment (xehu) [reply cypress.config.js]: Just resolved by reverting this file!

**Pull request merged**